src/components/organisms/WorldEntranceForm/WorldEntranceForm.tsx

```
@@ -90,25 +95,25 @@ export const WorldEntranceForm: React.FC<WorldEntranceFormProps> = ({
   });
 
   useEffect(() => {
-    if (!isEqual(world.questions?.profile, profileQuestionsCopy)) {
-      profileQuestionsCopy = world.questions?.profile;
+    if (!isEqual(world.questions?.profile, profileQuestionsList)) {
```

Maybe fail fast for more linear code:

```ts
if( isEqual() ) return;
```